overlord/servicestate/quota_control.go

```diff
@@ -313,6 +313,9 @@ func EnsureSnapAbsentFromQuota(st *state.State, snap string) error {
 	return nil
 }
 
+// AddSnapToQuotaGroup returns a task for adding a snap to a quota group. It wraps the task creation
+// with proper conflict detection for the affected quota-group. Conflict detection for the snap being
+// added must be done by the larger context, as this function is not intended to be run on its own.
```

Comment:
s/is not intended to be run on its own/is intended to be used in the context of a more complex change./

Reply:
thank you for the review, really appreciated. updated this!